test/accountant.ts

```diff
@@ -305,15 +305,15 @@ describe('Accountant', () => {
             stub.callCount.should.eq(claims.length);
         });
 
-        it('should allow undefined transactions', async () => {
-            const claims = defaultClaims();
-            const subject = new Accountant(undefined, claims, MinimumSenderBalance, client, logger);
+        // it('should allow undefined transactions', async () => {
```

Comment:
Is this test failing?

Reply:
Moved the `client` ( that requires a node running ) into the e2e-test folder, wdyt ? Is it better if I add the running node to the unit test and leave it as it is ?

Reply:
Regarding the question: I forgot to re enable it, thanks: enabled

Reply:
> Is it better if I add the running node to the unit test and leave it as it is ?

makes more sense in the e2e-test folder :+1: